CONTRIBUTING.md

```diff
@@ -1,17 +1,58 @@
 # Contributing
+To make contributions to this charm, you'll need a working Juju development setup.
 
-To make contributions to this charm, you'll need a working [Juju development setup](https://juju.is/docs/sdk/dev-setup).
+## Prerequisites
+### Charmcraft installation
+Charmcraft depends on LXD to build the charms in a container matching the target base(s).
 
-You can use the environments created by `tox` for development:
+To install Charmcraft and LXD run the following commands:
+```shell
+sudo snap install --classic charmcraft
+sudo snap install lxd
+sudo adduser $USER lxd
+newgrp lxd
+lxd init --auto
+```
+
+### MicroK8s installation
+To install MicroK8s run the following commands:
+```shell
+sudo snap install microk8s --channel=1.27-strict/stable
+sudo usermod -a -G snap_microk8s $USER
+newgrp snap_microk8s
+sudo microk8s enable hostpath-storage
+```
 
+### Install Juju
+To install Juju run the following command:
+```shell
+sudo snap install juju
+```
+To bootstrap a Juju controller on the MicroK8s instance run the following command:
+```shell
+juju bootstrap microk8s
+```
+
+### Install tox
+This project requires `tox>=4.0.0` for development and testing, which is available as a `pip` package.
+
+To install `pip` and `tox`, run the following commands:
+```shell
+sudo apt install python3-pip
+python3 -m pip install tox
```

Comment:
> Should LXD be in its own section?

In the SDK docs Charmcraft and LXD installation and init are on the same section. I would like to avoid having a lot of sections with a one-liner command in each.